src/import.ml

```diff
@@ -128,6 +128,8 @@ module Fmt = struct
   let list = Format.pp_print_list
   let string s ppf = Format.pp_print_string ppf s
 
+  let nl = Format.pp_print_newline
```

Comment:
I wonder whether it is worth defining this for a single use.

Reply:
I can find a few other use cases very fast if you'd like. I just refrained from touching unrelated bits of the code :)

Reply:
BTW, at some point we should try to get rid of this `Fmt` module and use `Stdune.Pp` everywhere

Reply:
OK, Sure. We can leave that for a later pr.